docs/dev_sessions/DEV_SESSIONS.md

```diff
@@ -1,3 +1,9 @@
+---
+order: 1
+parent:
+  title: Developer Sessions
```

Comment:
@fadeev I couldn't get this title to display in the sidebar, any ideas?

Reply:
`parent` is only parsed in `readme.md` at this point. Otherwise, if we have several files in a directory, and each has a `parent` property, the script wouldn't know which to use, hence, `readme.md`.

So, I've added readme.md:

```
---
order: false
parent:
  title: Developer Sessions
  order: 8
---

Reply:
ah okay, makes sense, thank you! 🔥 🚰 🚒 👨‍🚒 👩‍🚒

Reply:
I promise to write a readme on how to use this theme 😅 after we settle on the details.